R/read_vc.R

```diff
@@ -13,6 +13,19 @@ setGeneric(
   }
 )
 
+#' @rdname read_vc
+#' @importFrom methods setMethod
+setMethod(
+  f = "read_vc",
+  signature = signature(root = "ANY"),
+  definition = function(file, root){
+    if (missing(root)) {
+      stop("'root' is missing")
```

Comment:
the code below would make `"."` the default root. Can you apply it to `write_vc()` too?
```suggestion
      read_vc(file = file, root = ".")
```